src/views/Drawer/DrawerNavigatorItems.js

```diff
@@ -22,6 +22,7 @@ import type {
 
 type Props = {
   navigation: NavigationScreenProp<NavigationState, NavigationAction>;
+  routes?: Array<*>;
```

Comment:
As in use a type other than `*`?

The original code doesn't define a type for the items `navigation.state.routes.map((route: *, index: number) => {`.